src/modules/usage/usage.service.spec.ts

```diff
@@ -117,31 +152,73 @@ describe('UsageService', () => {
       const result = await service.addDailyUsageChangeOnFileSizeChange(
         user,
         oldFile,
-        yesterdayFile,
+        fileWithDecreasedSize,
       );
 
       expect(result).toEqual(expectedUsage);
       expect(usageRepository.create).toHaveBeenCalledWith(
         expect.objectContaining({
           userId: user.uuid,
-          delta: 100,
+          delta: -50,
           type: UsageType.Daily,
         }),
       );
     });
 
-    it('When file size decreased and not created today, then should create daily usage with negative delta', async () => {
-      const existingUsage = newUsage();
-      const yesterdayFile = newFile({
+    it('When file was created after the usage period, then should return null', async () => {
+      const usagePeriod = new Date('2024-06-15T00:00:00.000Z');
+      const dateAfterUsage = new Date('2024-06-20T00:00:00.000Z');
+
+      const existingUsage = newUsage({
         attributes: {
-          size: BigInt(50),
-          createdAt: new Date(Date.now() - 24 * 60 * 60 * 1000),
+          period: usagePeriod,
+          type: UsageType.Monthly,
+        },
+      });
+
+      const newFileData = newFile({
+        attributes: {
+          size: BigInt(200),
+          createdAt: dateAfterUsage,
         },
       });
+
+      jest
+        .spyOn(usageRepository, 'getMostRecentMonthlyOrYearlyUsage')
+        .mockResolvedValue(existingUsage);
+
+      const result = await service.addDailyUsageChangeOnFileSizeChange(
```

Comment:
This function may return void as it 'adds' things, otherwise the naming may be wrong